neighsyncd/neighsync.cpp

```diff
@@ -38,8 +39,12 @@ void NeighSync::onMsg(int nlmsg_type, struct nl_object *obj)
 
     key+= LinkCache::getInstance().ifindexToName(rtnl_neigh_get_ifindex(neigh));
     key+= ":";
-    nl_addr2str(rtnl_neigh_get_dst(neigh), addrStr, MAX_ADDR_SIZE);
-    key+= addrStr;
+
+    nl_addr2str(rtnl_neigh_get_dst(neigh), ipStr, MAX_ADDR_SIZE);
+    string ip(ipStr);
+    if (family == IPV6_NAME && ip.compare(0, 4, "ff02"))
```

Comment:
this depends on nl_addr2str impl, In the api, it says the Return is "Address represented in ASCII stored in destination buffer.", it does not mention it is already lowercase. https://www.infradead.org/~tgr/libnl/doc/api/group__addr.html#ga6471f04b9fa8ef1d4af1bf2b87b49529

I do not know if it is a reliable assumption.